core/src/main/scala/org/apache/spark/scheduler/ApplicationEventListener.scala

```diff
@@ -57,4 +58,10 @@ private[spark] class ApplicationEventListener extends SparkListener {
       adminAclsGroups = allProperties.get("spark.admin.acls.groups")
     }
   }
+
+  override def onOtherEvent(event:SparkListenerEvent):Unit = event match {
```

Comment:
oh ok I thought I had space before the style check complained about it but might have interpreted it wrongly